language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/RemoteMethodCallActionNodeContext.java

```diff
@@ -53,6 +53,14 @@ public RemoteMethodCallActionNodeContext() {
     public List<LSCompletionItem> getCompletions(BallerinaCompletionContext context, RemoteMethodCallActionNode node)
             throws LSCompletionException {
         List<LSCompletionItem> completionItems = new ArrayList<>();
+        if (onSuggestClients(node, context)) {
+            // Covers following:
+            // 1. cl<cursor>->func()
+            completionItems.addAll(this.expressionCompletions(context));
+            this.sort(context, node, completionItems);
```

Comment:
Shall we improve the sorting in this context as well
![image](https://user-images.githubusercontent.com/35211477/207663387-b0641042-3965-42b0-9dff-fb961613bf4a.png)

Reply:
Done. Addressed. Also addressed the same for client resource access action